internal/services/formatters/ruby/bundler/formatter.go

```diff
@@ -111,49 +110,44 @@ func (f *Formatter) removeOutputEsc(output string) string {
 }
 
 func (f *Formatter) parseOutput(output, projectSubPath string) {
-	if strings.Contains(output, "No vulnerabilities found") {
+	if strings.Contains(output, "No vulnerabilities found") || !strings.Contains(output, "Name:") {
```

Comment:
Why this is needed? Maybe add a comment to let ir more clear.

Reply:
This is the fix for the bug described on commit message.